src/Integration.Vsix/SonarLint.vsct

```diff
@@ -30,6 +30,9 @@
       <Group guid="guidCmdSet" id="SonarLintCommandSubMenu" priority="0x0100">
```

Comment:
Considering we will have a number of submenus eg. help, analysis and so on shouldn't we consider having a seperate vsct file for each? I have a feeling else this file will be super full and also it will be more difficult for us to simultaneously work on it.

Each of the vsct would need a reference to the guidCmdSet and the SonarLintCommandSubMenu so maybe those could be saved somewhere accessible instead of copy pasting the guids around. What do you think?

@rita-sonarsource and @duncanp-sonar